includes/discovery/bgp-peers/firebrick.inc.php

```diff
@@ -33,17 +33,17 @@
     if (strlen($address) > 15) {
         $address = IP::fromHexString($address)->compressed();
     }
-
-    if (isset($value['fbBgpPeerTableId'])) {
+    if (isset($value['fbBgpPeerTableId']) && $value['fbBgpPeerTableId'] !== '') {
         $bgpPeers[$value['fbBgpPeerTableId']][$address] = $value;
     } else {
-        $bgpPeers[0][$address] = $value;
+        $bgpPeers[null][$address] = $value;
     }
 }
 unset($bgpPeersCache);
 
 foreach ($bgpPeers as $vrfId => $vrf) {
     if (empty($vrfId)) {
+        $vrfId = null;
```

Comment:
not necessary cause this variable is not used after the empty() test above

Reply:
This was in use in the insert - and it was that that was causing part of the issues.  I've put a comment in accordingly.